action_access_control_list/models/ir_action_access.py

```diff
@@ -4,64 +4,57 @@
 
 from odoo import api, models, fields
 
-class IrActionAccess(models.Model):
 
-    """
-    Actions access rule
-    """
+class IrActionAccess(models.Model):
+    """Actions Access Rule"""
 
     _name = 'ir.action.access'
+    _description = __name__
 
     group_id = fields.Many2one(
-        string='Group',
-        comodel_name='res.groups',
+        'res.groups',
```

Comment:
@ventilooo Aesthetic changes should be avoided unless it's made to comply with an already established projet style.

Reply:
@joaoalf this is my change. The module is still not in production. This was done only to make the code more standard.